src/robotide/controller/commands.py

```diff
@@ -279,12 +280,35 @@ class NullObserver(object):
 class RenameKeywordOccurrences(_ReversibleCommand):
 
     def __init__(self, original_name, new_name, observer, keyword_info=None):
-        self._original_name = original_name
-        self._new_name = new_name
+        keyword_name, new_keyword_name = self._check_gherkin(new_name,
```

Comment:
These local variables are redundant. You can do `self._original_name, self._new_name = self._check_gherkin(...)`